src/librepgp/stream-dump.cpp

```diff
@@ -130,7 +130,7 @@ static pgp_map_t key_type_map[] = {
 };
 
 static pgp_map_t pubkey_alg_map[] = {
-  {PGP_PKA_RSA, "RSA"},
+  {PGP_PKA_RSA, "RSA (Encrypt or Sign)"},
```

Comment:
I just want to point out that we should keep in mind that when we change the JSON packet dumper we are effectively breaking our public interface. If I had an application using `rnp_dump_packets_to_json` that assumed this value was stable, it would be pretty easy to imagine it crashing/misbehaving/failing when upgrading to a newer version of rnp with the change in this PR.

Having said that, it seems like this change would make things more consistent, right? Maybe we should just document that the output of `rnp_dump_packets_to_json` is subject to change?

Reply:
@dewyatt agree, we should not make breaking changes into the JSON output. But somehow we should handle the case when we need to fix some older error (like this PR does).
So in this particular case I think we may let it go, but later on try to avoid it as much as possible - anyway we'll need to update it with new signature subpackets, algorithms, v5 fingerprint and so on so on.

Reply:
Well apparently this does break ruby-rnp's tests, so we'll need to figure out what to do about that.

Reply:
I'll go ahead and put in a workaround for ruby-rnp's tests and then hopefully we'll be all green here.

Reply:
Thank you @dewyatt !

Reply:
Thanks, @dewyatt !